microsoft-edge/visual-studio-code/index.md

```diff
@@ -20,17 +20,11 @@ To acquire any of the extensions highlighted below, navigate to Extensions \(sel
 
 Search the Marketplace for the specific extension and choose **Install**.  
```

Comment:
change:
navigate to Extensions
to:
navigate to **Extensions**
since it's a UI item to select.